src/main/frontend/handler/repo.cljs

```diff
@@ -364,10 +327,11 @@
                               (and (or (string/includes? (str error) "401")
                                        (string/includes? (str error) "404"))
                                    (not fallback?))
-                              (request-app-tokens!
-                               (fn []
-                                 (pull repo-url (state/get-github-token repo-url) {:fallback? true}))
-                               nil)
+                              (p/let [token (helper/get-github-token repo-url)]
+                               (helper/request-app-tokens!
```

Comment:
I think there's no need to call `helper/request-app-tokens!` here since you already had the `token`.

Reply:
I think we can remove token parameters from the `poll` function and get the token from the `poll` function internal.